build.gradle

```diff
@@ -17,20 +17,15 @@ buildscript {
         classpath "com.netflix.nebula:gradle-aggregate-javadocs-plugin:3.0.1"
         classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.8"
         classpath "io.franzbecker:gradle-lombok:3.2.0"
-        /*
-         * TODO: Upgrading to 1.3.1 caused problems
-         * A problem was found with the configuration of task ':genie-ui:npmInstall'.
-         * > File '{somePath}/genie/genie-ui/package-lock.json' specified for property '$2' does not exist.
-         */
-        classpath "com.moowork.gradle:gradle-node-plugin:1.2.0"
+        classpath "com.netflix.nebula:nebula-node-plugin:1.3.0"
```

Comment:
This is a fork of original `gradle-node-plugin` which is not maintained anymore and it is not compatible with Gradle 6.

Reply:
Will this be maintained or should we be looking into some other options?

Reply:
Yes, we plan to maintain compatibility. We don't plan any feature development.